context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -46,6 +52,11 @@ def build_vitessce_conf(self):
             ]
         )
         found_images = get_matches(file_paths_found, full_seqfish_reqex)
+        if len(found_images) == 0:
+            message = f'seqFish assay with uuid {self._uuid} has no matching files'
+            if not self._is_mock:
+                current_app.logger.info(message)
+            raise FileNotFoundError(message)
```

Comment:
```suggestion
            raise FileNotFoundError(f'seqFish assay with uuid {self._uuid} has no matching files')
```
Let `return_empty_json_if_error` do the logging